packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json

```diff
@@ -265,8 +265,8 @@
           "Ref": "EcsCluster97242B84"
         },
         "DeploymentConfiguration": {
-          "MaximumPercent": 200,
-          "MinimumHealthyPercent": 50
+          "maxPercent": 200,
```

Comment:
This seems wrong. Was this a bug? I really didn’t expect the CFN to change just the construct APi

Reply:
I didn't mean to make any structure change. Just changed all the variable names containing maximum -> max and minimum -> min.

Reply:
I just want to change the variable names and populate the change to all of their usages. Will that be a problem if I don't make changes to this test files?

Reply:
You cannot change the names in the `expected` files, as they are defined by the CloudFormation Spec. Why do you want to rename them anyway?

Reply:
Thanks for the feedback. I might need to talk with the team to see if they still want to change those names.

Reply:
Here's the thing: you DO want to change the names of the parameters in CDK structs, but you CAN'T change what names they generate to in the CloudFormation template.

Reply:
> Here's the thing: you DO want to change the names of the parameters in CDK structs, but you CAN'T change what names they generate to in the CloudFormation template.

Yeah. I agree with you. I tried to revert the change that I did to the json template, but the unit tests failed.